Library/PhotosController.swift

```diff
@@ -48,9 +47,31 @@ class PhotosController: UICollectionViewController {
             self.sections = Photo.constructRemoteElements()
             self.collectionView?.reloadData()
         }
+
+        #if os(tvOS)
+        let playPauseTapRecognizer = UITapGestureRecognizer(target: self, action: #selector(self.playPause(gesture:)))
+        playPauseTapRecognizer.allowedPressTypes = [NSNumber(value: UIPressType.playPause.rawValue)]
+        self.collectionView?.addGestureRecognizer(playPauseTapRecognizer)
+        #endif
     }
 
     #if os(tvOS)
+    func playPause(gesture: UITapGestureRecognizer) {
```

Comment:
Oke s here you start playing the slideShow right? From the focussed cell onwards. Im just wondering how it will pause in this method...